examples/consul-examples-helper/consul-examples-helper.sh

```diff
@@ -152,16 +152,20 @@ function get_consul_cluster_ips {
 
 function print_instructions {
   local readonly server_ips=($@)
-  local readonly server_ip="${server_ips[0]}"
+  local readonly server_ip_0="${server_ips[0]}"
+  local readonly server_ip_1="${server_ips[1]}"
+  local readonly server_ip_2="${server_ips[2]}"
 
   local instructions=()
   instructions+=("\nYour Consul servers are running at the following IP addresses:\n\n${server_ips[@]/#/    }\n")
   instructions+=("Some commands for you to try:\n")
-  instructions+=("    consul members -http-addr=$server_ip:8500")
-  instructions+=("    consul kv put -http-addr=$server_ip:8500 foo bar")
-  instructions+=("    consul kv get -http-addr=$server_ip:8500 foo")
+  instructions+=("    consul members -http-addr=$server_ip_0:8500")
+  instructions+=("    consul kv put -http-addr=$server_ip_0:8500 foo bar")
+  instructions+=("    consul kv get -http-addr=$server_ip_0:8500 foo")
+  instructions+=("    consul kv get -http-addr=$server_ip_1:8500 foo")
+  instructions+=("    consul kv get -http-addr=$server_ip_2:8500 foo")
```

Comment:
Seems more sane to do this in a for-loop, rather than a hard-coded list of 3 servers.